src-ts/tools/work/work-detail-details/work-detail-details-pane/WorkDetailDetailsPane.tsx

```diff
@@ -1,115 +1,64 @@
+import { Link } from '@reach/router'
 import _ from 'lodash'
-import { FC } from 'react'
+import { Dispatch, FC, SetStateAction, useEffect, useState } from 'react'
+
+import { LoadingSpinner } from '../../../../lib'
+import { workFactoryMapFormData } from '../../../../lib/work-provider/work-functions/work-factory'
```

Comment:
we should not be importing anything from directories w/in the lib. the work factory should export the data and that will be automatically be available in the lib barrel file.